### PR TITLE
fix: remove group owner concept

### PR DESCRIPTION
**Pull request by nunogois**
https://linear.app/unleash/issue/2-140/adapt-ui-to-no-longer-have-the-group-owner-concept

### CODE REVIEW

**Comment by sighphyre** [comment]
This depends on: https://github.com/Unleash/unleash/pull/1905 which is probably why the group tests are failing

**Review by sjaanus — APPROVED** [review]
LGTM!

File: src/component/admin/groups/CreateGroup/CreateGroup.tsx [hunk 1]
```diff
@@ -62,6 +67,17 @@ export const CreateGroup = () => {
         navigate(GO_BACK);
     };
 
+    const onSetName = (name: string) => {
+        clearErrors();
+        if (groups?.filter(group => group.name === name).length) {
```
Comment (FredrikOseberg) [inline src/component/admin/groups/CreateGroup/CreateGroup.tsx:72]: Could extract to a variable for clarity. Now you need to read and understand the expression to know what happens here. Makes it hard to scan the code.

File: src/component/admin/groups/EditGroup/EditGroup.tsx [hunk 2]
```diff
@@ -65,6 +70,22 @@ export const EditGroup = () => {
         navigate(GO_BACK);
     };
 
+    const onSetName = (name: string) => {
+        clearErrors();
+        if (
+            groups?.filter(group => group.name === name && group.id !== groupId)
```
Comment (FredrikOseberg) [inline src/component/admin/groups/EditGroup/EditGroup.tsx:76]: Could be extracted for clarity

**Review by FredrikOseberg — APPROVED** [review]
Looks good. Made some minor suggestions for clarity.

File: src/component/admin/groups/CreateGroup/CreateGroup.tsx (hunk 1 above)
Comment (nunogois) [reply src/component/admin/groups/CreateGroup/CreateGroup.tsx]: Addressed in https://github.com/Unleash/unleash-frontend/pull/1210/commits/84c6a37dcbe94b9eb34951200484f07fa3345cd5

File: src/component/admin/groups/EditGroup/EditGroup.tsx (hunk 2 above)
Comment (nunogois) [reply src/component/admin/groups/EditGroup/EditGroup.tsx]: Addressed in https://github.com/Unleash/unleash-frontend/pull/1210/commits/84c6a37dcbe94b9eb34951200484f07fa3345cd5

**Pull request merged**